common/src/main/java/dev/ftb/mods/ftbchunks/data/FTBChunksTeamData.java

```diff
@@ -181,7 +182,7 @@ public ClaimResult unclaim(CommandSourceStack source, ChunkDimPos pos, boolean c
 
 		if (chunk == null) {
 			return ClaimResults.NOT_CLAIMED;
-		} else if (chunk.teamData != this && !source.hasPermission(2) && !source.getServer().isSingleplayer()) {
+		} else if ((chunk.teamData != this && !source.hasPermission(2) && !source.getServer().isSingleplayer()) || (chunk.ownerUnclaim() && !team.getOwner().equals(Objects.requireNonNull(source.getPlayer()).getUUID()))) {
```

Comment:
If I'm reading this right, an ownership check is done even if `player.hasPermissions(2)`.  Op-level players should always be able to run this command.

Also, throwing an NPE here is not a good idea - `source.getPlayerOrException()` is more appropriate. But still not suitable here, since there are paths into this method via a command run on the console.